tuxemon/db.py

```diff
@@ -35,15 +35,7 @@
 import os
 from enum import Enum
 from operator import itemgetter
-from typing import (
-    Any,
-    Dict,
-    Literal,
-    Mapping,
-    Optional,
-    Sequence,
-    overload,
-)
+from typing import Any, Dict, Literal, Mapping, Optional, Sequence, overload
```

Comment:
Picky, but should this have changed? Could you run isort and black against this file? (Just run script/fmt.sh and only commit the change to files that are currently in this PR).

If this causes lots of unrelated changes then let me know.

Reply:
Looks good other than that.